contracts/Standards/IContractId.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity 0.4.24;
+
+/// @title uniquely identifies deployable (non-abstract) platform contract
+/// @notice cheap way of assigning implementations to knownInterfaces which represent system services
+///         unfortunatelly ERC165 does not include full public interface (ABI) and does not provide way to list implemented interfaces
+///         EIP820 still in the making
+/// @dev ids are generated as follows keccak256("neufund-platform:<contract name>")
+///      ids roughly correspond to ABIs
+contract IContractId {
```

Comment:
Do all deployed contracts get this?

Reply:
Just curious

Reply:
all new contracts got it. the ones already deployed like Neumark of course not. btw. I just realized that IdentityRegistry didn't get it. I'm adding this now + tests